app-libs/stf/src/cli.rs

```diff
@@ -330,6 +330,113 @@ pub fn cmd<'a>(
 					Ok(())
 				}),
 		)
+		.add_cmd(
+			Command::new("play-turn")
+				.description("Player turn of connect four")
+				.options(|app| {
+					app.setting(AppSettings::ColoredHelp)
+						.arg(
+							Arg::with_name("player")
+								.takes_value(true)
+								.required(true)
+								.value_name("SS58")
+								.help("player's AccountId in ss58check format"),
+						)
+						.arg(
+							Arg::with_name("column")
+								.takes_value(true)
+								.required(true)
+								.value_name("u8")
+								.help("play stone in column, must be in the range of [0,7]"),
```

Comment:
there are 7 columns: range [0,6]?

Reply:
Hm. I see. Then the check in the connect four is not appropriate: https://github.com/ajuna-network/ajuna-node/blob/update-substrate-5/pallets/ajuna-connectfour/src/lib.rs#L288

Should be [1, 7] then.

Reply:
Updated, thanks for the review!